### ISSUE TITLE
Amplify Video Community Site

### ISSUE THREAD

**Issue by smp** [awslabs/amplify-video#290]
We'd like your feedback and comments on a new community-focused site for Amplify Video. 

**Goals**

This site would serve many purposes, but the three primary goals are as follows..

**1. Educational Index & Community News** - We're seeing a tremendous amount of Amplify Video activity on this github project, youtube, AWS blogs, and other sites, however, there doesn't yet exist a landing site for the project that can serve as an index to these resources. This community site would provide a way for community members to post demos, blogs, and videos, related to amplify video and have it seen by others interested in the project.
**2. Documentation** - Github wiki leaves a lot to be desired in terms of searchability, maintenance (no PRs?), and discoverability. This site would become the new landing page for the project and allow us to index other sub-projects like a new docs site.
**3. Dogfood & Meta Content** - By building and maintaining a community site together using Amplify and Amplify Video, we'll learn a lot about deploying and maintaining a site. Those learnings will help us create a better plugin, but also help us to create content that is about using Amplify Video in a greater context alongside other AWS services (search, recommendations, analytics) or open source projects that are combined to deliver a web-based media experience.

**Design & Contributions**

The design style of the site should match closely (or even be a fork of) the [Amplify Community Site](https://github.com/aws-amplify/community). We should also solicit contributions in a similar manner, but also add a section for contribution of videos, both hosted/processed natively by the site, or embedded from social locations like Youtube and Twitch.

**Community Questions**

- Do you support the idea of a site like this? If yes/no - why?
- What content would you like to see created in this new site?
- If you've created Amplify Video related content, would you be comfortable linking to it from this site? 

**FAQ**

Q: Why not just add your content to the existing Amplify Community Site?
A: We want to show others how to build, develop, and grow their own Amplify plugins. By having our own site dedicated to the Amplify Video project, we'll have 1) Complete control of how to develop and maintain the community site. 2) A site with focus on media application development within the Amplify Framework. 3) We'll be able respond quickly to the needs of our community site without having to consider the impact of site changes on the larger Amplify Community site.

**Comment by kylekirkby** [comment]
> Do you support the idea of a site like this? If yes/no - why?

Yes, love this idea! It would be awesome to have all the amplify-video resources in one place.

> What content would you like to see created in this new site?

1. A really well-put-together documentation site. Something similar to, https://docs.amplify.aws/ would be awesome! 
2. Blogs and a way to submit blogs.

> If you've created Amplify Video related content, would you be comfortable linking to it from this site?

Definitely! 💯
